packages/playground/src/utils/strings.ts

```diff
@@ -9,11 +9,21 @@ export interface GenerateNameOptions {
   suffix?: string;
 }
 export function generateName(length = 12, options?: GenerateNameOptions): string {
```

Comment:
I think the changes done here are not needed as we only need to adjust the length where we call the function.